Resources/config/services.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Ruvents\FormWizardBundle\EventListener\StepNotFoundExceptionListener;
+use Ruvents\FormWizardBundle\Storage\SessionStorage;
+use Ruvents\FormWizardBundle\Storage\StorageInterface;
+use Ruvents\FormWizardBundle\Type\TypeFacadeFactory;
+use Ruvents\FormWizardBundle\WizardFactory;
+use Ruvents\FormWizardBundle\WizardFactoryInterface;
+
+return function (ContainerConfigurator $container) {
+    $services = $container->services()
+        ->defaults()
+        ->private();
+
+    $services->set(TypeFacadeFactory::class)
+        ->args([
+            '$normalizer' => ref('serializer'),
+            '$denormalizer' => ref('serializer'),
+            '$validator' => ref('validator'),
+            '$formFactory' => '@form.factory',
```

Comment:
Not sure if this is the right reference. Unless I'm missing something, shouldn't this be `ref('form.factory')`?

Reply:
I need to write tests immediately :)

Reply:
@iltar , thank you!